boards/airmind/mindpx-v2/default.cmake

```diff
@@ -35,8 +35,8 @@ px4_add_board(
 		#lights/rgbled_ncp5623c
 		magnetometer # all available magnetometer drivers
 		mkblctrl
-		#optical_flow # all available optical flow drivers
-		optical_flow/px4flow
+		optical_flow # all available optical flow drivers
+		#optical_flow/px4flow
```

Comment:
This line can be removed now.

Reply:
thanks @TSC21 

Resolved as suggested.